src/sql/workbench/browser/modelComponents/groupContainer.component.ts

```diff
@@ -95,6 +97,10 @@ export default class GroupContainer extends ContainerBase<GroupLayout, GroupCont
 		return this.getPropertyOrDefault<boolean>((props) => props.collapsed, false);
 	}
 
+	public get header(): string {
+		return this._containerLayout?.header;
```

Comment:
Have default value so you aren't returning undefined.

Reply:
If you could spend a few minutes looking into why strict nulls didn't catch this either that'd be great...

Reply:
@Charles-Gagnon not super familiar with how this is determined but I think it is determined by this. https://github.com/microsoft/azuredatastudio/blob/2bc6a0cd012de69504061a876070cfa43533fedc/src/tsconfig.base.json#L12

Reply:
Extensions have their own - https://github.com/microsoft/azuredatastudio/blob/2bc6a0cd012de69504061a876070cfa43533fedc/extensions/tsconfig.base.json#L26

And that answers my question (which I thought I looked at but whoops) - since each extension then extends this for their own tsconfig and you can see that mssql has it disabled

https://github.com/microsoft/azuredatastudio/blob/2bc6a0cd012de69504061a876070cfa43533fedc/extensions/mssql/tsconfig.json#L2

A great fundamentals task if you're looking for something to do...